docs/releases/4.0.txt

```diff
@@ -601,6 +604,20 @@ Note that the default :file:`settings.py` file created by
 You can set ``USE_TZ`` to ``False`` in your project settings before then to
 opt-out.
 
+.. _use_l10n_deprecation:
+
+Localization
+------------
+
+In order to follow good practice, the default value of the ``USE_L10N`` setting
+is changed from ``False`` to True.
+
+Moreover ``USE_L10N`` is deprecated as of this release. Starting with Django
+5.0, by default, any date or number displayed by Django will be localized.
```

Comment:
Should we mention that existing values of `DECIMAL_SEPARATOR` and `THOUSAND_SEPARATOR` will now affect template rendering in all cases? The use of the settings for form validation only is documented, see ticket-22654 and 504e778.

I'm thinking about a user who reads the release note and thinks "good, let's have the locale defaults" but will get not the defaults but rather the settings they were previously only using for form validation.

Reply:
This is not something that we changed in this PR. I was thinking about changing this in Django 5.0 when `USE_L10N` will be removed :thinking: .

Reply:
Right, yes, 5.0 is a fine time to mention it, then.